templates/server.conf.erb

```diff
@@ -55,6 +65,8 @@ ssl_certfile: <%= scope['pulp::db_ssl_certfile'] %>
 verify_ssl: <%= scope['pulp::db_verify_ssl'] %>
 ca_path: <%= scope['pulp::db_ca_path'] %>
 <% end -%>
+# unsafe_autoretry: false
+# write_concern: majority
```

Comment:
These should be added as parameters, otherwise there will be no way to configure them (because puppet would overwrite any manual changes to this file).